cmd/fluxd/main.go

```diff
@@ -145,11 +144,17 @@ func main() {
 		dryGC        = fs.Bool("sync-garbage-collection-dry", false, "only log what would be garbage collected, rather than deleting. Implies --sync-garbage-collection")
 		syncState    = fs.String("sync-state", fluxsync.GitTagStateMode, fmt.Sprintf("method used by flux for storing state (one of {%s})", strings.Join([]string{fluxsync.GitTagStateMode, fluxsync.NativeStateMode}, ",")))
 
-		// registry
+		// registry scanning cache backend
+		cacheBackend = fs.String("cache-backend", "memcached", "select the registry cache backend (supported: memcached, redis)")
```

Comment:
Since these `"memcached", "redis"` are used in a switch below, I think it's worth having consts for them.

Reply:
fixed by 6767d842